app/models/dashboards/overview.rb

```diff
@@ -22,7 +28,7 @@ def terminated(start_date: @start_date, end_date: @end_date)
 
   def match_results(start_date: @start_date, end_date: @end_date)
     hash = terminated(start_date: start_date, end_date: end_date).group_by(&:first)
-    hash.merge(hash) { |key, ids| ids.map { |key, id| id }}
+    hash.merge(hash) { |_, ids| ids.map { |_, id| id } }
```

Comment:
I'm really confused last to what this is doing.  Is this equivalent to `ids.map(&:self)`? this is weird.

Reply:
After some exploration -- I determined that this is a particularly opaque version of
```
      group_by(&:shift).
      transform_values(&:flatten)
```